.circleci/config.yml

```diff
@@ -369,6 +369,19 @@ jobs:
           name: "Restore CCache cache"
           keys:
             - velox-ccache-debug-{{ arch }}-{{ checksum "merge-base-date" }}
+      - run:
+          name: "Install Protobuf 3.21.4"
```

Comment:
can we invoke `setup-ubuntu.sh install_protobuf` ?

Reply:
No, The docker image of circleCI is centos, and there is some "apt" instruction at setup-ubuntu.sh.